server/persistence/login.go

```diff
@@ -14,6 +15,22 @@ import (
 	"github.com/offen/offen/server/keys"
 )
 
+func (p *persistenceLayer) LoginSSO(email, salt string) (LoginResult, error) {
```

Comment:
The method receives a salt parameter, but it's not used from what I understand. Did you intend to use it when hashing the given email?

Reply:
That is indeed correct. I'll fix it in a bit :)